recipes/bagit-profile/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "bagit-profile" %}
+{% set version = "1.3.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bagit_profile-{{ version }}.tar.gz
+  sha256: 57798cdcf98b32a413edb29382d85f4f8c44d3204940d7e12d84998521a98c3f
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python =>3
```

Comment:
Shouldn|t it be >= instead of =>?

Reply:
Whoops! Not sure why it linted ok. Anyway I fixed it.